CMakeLists.txt

```diff
@@ -35,23 +99,23 @@ if (BUILD_TESTING)
       SZ_STACK=1
   )
   configure_file(
-    StabilizerToolchain.cmake.in
-    StabilizerToolchain.cmake.in
+    ${CMAKE_CURRENT_SOURCE_DIR}/cmake/StabilizerToolchain.cmake.in
+    ${CMAKE_CURRENT_BINARY_DIR}/cmake/StabilizerToolchain.cmake.tmp
     @ONLY
   )
   file(GENERATE
-    OUTPUT StabilizerToolchain.cmake
-    INPUT ${CMAKE_CURRENT_BINARY_DIR}/StabilizerToolchain.cmake.in
+    OUTPUT ${CMAKE_CURRENT_BINARY_DIR}/cmake/StabilizerToolchain.cmake
+    INPUT ${CMAKE_CURRENT_BINARY_DIR}/cmake/StabilizerToolchain.cmake.tmp
   )
 
   # Add tests project
   include(ExternalProject)
   ExternalProject_Add(
     tests
     DEPENDS LLVMStabilizer stabilizer
-    SOURCE_DIR ${CMAKE_SOURCE_DIR}/tests
+    SOURCE_DIR ${CMAKE_CURRENT_SOURCE_DIR}/tests
```

Comment:
I think here we can omit `CMAKE_CURRENT_SOURCE_DIR` too.

Reply:
I am more tempted to keep this for consistency actually, even if it does not alter the behavior.

Reply:
OK, I'm fine with that.